cpp/dolfinx/fem/CoordinateElement.cpp

```diff
@@ -5,6 +5,8 @@
 // SPDX-License-Identifier:    LGPL-3.0-or-later
 
 #include "CoordinateElement.h"
+#include "FiniteElement.h"
+#include <iostream>
```

Comment:
I guess this is a leftover include from debugging?

Reply:
Yes, I will remove